code/modules/clothing/clothing.dm

```diff
@@ -649,63 +696,48 @@
-			M.matchignite()
+/obj/item/clothing/shoes/item_interaction(mob/living/user, obj/item/used, list/modifiers)
+	if(istype(used, /obj/item/match) && src.loc == user)
+		var/obj/item/match/match = used
+		if(!match.lit && !match.burnt) // Match isn't lit, but isn't burnt.
+			user.visible_message("<span class='warning'>[user] strikes a [match] on the bottom of [src], lighting it.</span>","<span class='warning'>You strike [match] on the bottom of [src] to light it.</span>")
+			match.matchignite()
 			playsound(user.loc, 'sound/goonstation/misc/matchstick_light.ogg', 50, 1)
-			return
-		if(M.lit && !M.burnt)
-			user.visible_message("<span class='warning'>[user] crushes [M] into the bottom of [src], extinguishing it.</span>","<span class='warning'>You crush [M] into the bottom of [src], extinguishing it.</span>")
-			M.dropped()
+			return ITEM_INTERACT_SUCCESS
+		if(match.lit && !match.burnt)
+			user.visible_message("<span class='warning'>[user] crushes [match] into the bottom of [src], extinguishing it.</span>","<span class='warning'>You crush [match] into the bottom of [src], extinguishing it.</span>")
+			match.dropped()
 		return
 
-	if(istype(I, /obj/item/wirecutters))
-		if(can_cut_open)
-			if(!cut_open)
-				playsound(src.loc, I.usesound, 100, 1)
-				user.visible_message("<span class='warning'>[user] cuts open the toes of [src].</span>","<span class='warning'>You cut open the toes of [src].</span>")
-				cut_open = TRUE
-				update_appearance(UPDATE_NAME|UPDATE_DESC|UPDATE_ICON_STATE)
-			else
-				to_chat(user, "<span class='notice'>[src] have already had [p_their()] toes cut open!</span>")
-		return
-
-	if(istype(I, /obj/item/kitchen/knife/combat))
+	if(istype(used, /obj/item/kitchen/knife/combat))
 		if(!knife_slot)
-			to_chat(user, "<span class='notice'>There is no place to put [I] in [src]!</span>")
+			to_chat(user, "<span class='notice'>There is no place to put [used] in [src]!</span>")
 			return
 		if(hidden_blade)
 			to_chat(user, "<span class='notice'>There is already something in [src]!</span>")
 			return
-		if(!user.unEquip(I))
+		if(!user.unEquip(used))
 			return
-		user.visible_message("<span class='notice'>[user] places [I] into their [name]!</span>", \
-			"<span class='notice'>You place [I] into the side of your [name]!</span>")
-		I.forceMove(src)
-		hidden_blade = I
-		return
-
+		user.visible_message("<span class='notice'>[user] places [used] into their [name]!</span>", \
+			"<span class='notice'>You place [used] into the side of your [name]!</span>")
+		used.forceMove(src)
+		hidden_blade = used
+		return ITEM_INTERACT_SUCCESS
 	return ..()
 
+/obj/item/clothing/shoes/wirecutter_act(mob/living/user, obj/item/I)
+	if(can_cut_open)
+		if(cut_open)
+			to_chat(user, "<span class='notice'>[src] have already had [p_their()] toes cut open!</span>")
+			return
+		if(!I.use_tool(src, user, 0, volume = I.tool_volume))
+			return
```

Comment:
These returns should also probably return TRUE because even if the thing we're trying to do failed to happen, we still don't want the rest of the attack chain to be attempted. Otherwise you might cause an attack after attempting to use the wirecutters on shoes that are already cut open.